portal/src/main/webapp/WEB-INF/jsp/global/global_variables.jsp

```diff
@@ -131,12 +131,42 @@
 <!-- Global variables : basic information about the main query -->
 <script type="text/javascript">
 
+    // these are new
+    
+    window.serverVars = {
```

Comment:
thesea are consumed by frontend app.  Frontend app no longer needs QuerySession

Reply:
I like this. I was thinking maybe we should update the `.github/PULL_REQUEST_TEMPLATE.md` for these type of conventions. Like add a checkbox to the template
- [ ] If this PR passes variables from the server to frontend, put them in `window.serverVars`